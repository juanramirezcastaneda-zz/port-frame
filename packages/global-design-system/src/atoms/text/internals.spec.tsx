describe('Text', () => {
  it('Should render a text component with the corresponding font-size', () => {
    expect(true).toBe(true);
  });
});
