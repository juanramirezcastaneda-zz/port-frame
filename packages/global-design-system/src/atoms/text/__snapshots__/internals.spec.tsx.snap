// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Text should render a text component as expected 1`] = `
<span
  className="sc-bdVaJa ioNgrB"
  size="100"
>
  fakeText
</span>
`;
